lib/spack/spack/package_prefs.py

```diff
@@ -25,23 +25,7 @@ def _spec_type(component):
 
 def get_packages_config():
     """Wrapper around get_packages_config() to validate semantics."""
```

Comment:
The docstring seems outdated as we are not validating anything in the new version. Wondering if we should keep the function or inline it. In case we keep it we can mark it as private with a leading underscore.

Reply:
I'll just inline it, it's a single function call now.